docs/tuning-apps/networking/ofi.rst

```diff
@@ -76,6 +78,11 @@ list. For example:
 
    shell$ ompi_info --param mtl ofi --level 9
 
+.. note:: When using the HPE CXI provider and ``mpirun`` as the job launcher,
```

Comment:
Do you want to make this `.. important::`?

FWIW: here's all the possibilities of colored boxes: https://docs.open-mpi.org/en/v5.0.x/developers/rst-for-markdown-expats.html#brightly-colored-boxes